electrum/wizard.py

```diff
@@ -704,14 +704,17 @@ def do_configure_server(self, wizard_data: dict):
         self._logger.debug(f'configuring server: {wizard_data!r}')
         net_params = self._daemon.network.get_parameters()
         server = ''
+        oneserver = wizard_data['one_server']
         if not wizard_data['autoconnect']:
             try:
                 server = ServerAddr.from_str_with_inference(wizard_data['server'])
                 if not server:
                     raise Exception('failed to parse server %s' % wizard_data['server'])
             except Exception:
                 return
-        net_params = net_params._replace(server=server, auto_connect=wizard_data['autoconnect'])
+        else:
+            oneserver = False
```

Comment:
Can we avoid duplicating the default value here?
Would this work?
```suggestion
            oneserver = net_params.oneserver 
```

Reply:
That was indeed the intention :)

I added a patch.
Note that it's still possible to `setconfig` both options to `true`, but it will honor `oneserver` over `auto_connect` at startup and at changing network parameters.